packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -19,6 +19,7 @@ import { I18nextProvider } from "react-i18next";
 import { getTypedObjectKeys } from "./getTypedObjectKeys";
 
 export type OdysseyI18nResourceKeys = (typeof resources)["en"];
+export const OdysseyI18nResourceKeysList = getTypedObjectKeys(resources["en"]);
```

Comment:
This feels a little weird being capitalized since it's not a type or component. Feels like we should camel case it to me
```suggestion
export const odysseyI18nResourceKeysList = getTypedObjectKeys(resources["en"]);
```